cdap-api/src/main/java/co/cask/cdap/api/flow/flowlet/AbstractFlowlet.java

```diff
@@ -106,6 +110,70 @@ protected void setProperties(Map<String, String> properties) {
   }
 
   /**
+   * @see FlowletConfigurer#addStream(Stream)
+   */
+  protected final void addStream(Stream stream) {
```

Comment:
actually, does it make sense to be able to add a stream in a flowlet? Connections between streams and flowlets has to happen at the flow level right?